src/core/server/pulse/channel.ts

```diff
@@ -35,8 +35,9 @@ export interface ChannelConfig {
   logger: Logger;
 }
 export interface ChannelSetupContext {
-  elasticsearch: IClusterClient;
-  savedObjects: SavedObjectsServiceSetup;
+  pulse?: IPulseClient;
+  elasticsearch?: IPulseElasticsearchClient;
```

Comment:
Should we keep on using the Elasticsearch client wrapper or simply send the elasticsearch client from core?

Reply:
The Elasticsearch client wrapper exposes some methods that we want to reuse (e.g. `buildIndex` and `createIndexIfNotExist`. These prevent duplicating code and encourage us to use the same implementation across different pulse channels. I think we should keep on using the wrapper rather than using elasticsearch from core.